media/server/service/source/main.cpp

```diff
@@ -26,15 +26,23 @@
 
 int main(int argc, char *argv[])
 {
-    const char commitID[] = COMMIT_ID;
+    const char kSrcRev[] = SRCREV;
+    const char kTags[] = TAGS;
 
-    if (std::strlen(commitID) > 0)
+    if (std::strlen(kSrcRev) > 0)
     {
-        RIALTO_SERVER_LOG_MIL("Commit ID: %s", commitID);
+        if (kTags[0] == 'v')
```

Comment:
This might crash if kTags will be empty. Replace it with std::strlen(kTags) > 0. You already check the string pattern in CMake.